infra/gcp/terraform/kubernetes-public/k8s-metrics.tf

```diff
@@ -0,0 +1,93 @@
+/*
+This file defines:
+- bigquery dataset for triage to store temp results
+- GCS bucket to serve go.k8s.io/triage results
+- IAM bindings
```

Comment:
Copy-pasta comments need updating

I also think we need to put boilerplate license in our tf files...